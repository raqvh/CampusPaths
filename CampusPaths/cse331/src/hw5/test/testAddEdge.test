# Tests method AddEdge for a graph

# Create an empty graph
CreateGraph graph1

# Add nodes
AddNode graph1 n1
AddNode graph1 n2

# Add edge
AddEdge graph1 n1 n2 e1

# Print edges
ListChildren graph1 n1
