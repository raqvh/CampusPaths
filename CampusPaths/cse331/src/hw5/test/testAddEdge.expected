# Tests method AddEdge for a graph

# Create an empty graph
created graph graph1

# Add nodes
added node n1 to graph1
added node n2 to graph1

# Add edge
added edge e1 from n1 to n2 in graph1

# Print edges
the children of n1 in graph1 are: n2